repository_test.go

```diff
@@ -902,8 +902,13 @@ func (s *RepositorySuite) TestPlainOpenNotExistsDetectDotGit() {
 }
 
 func (s *RepositorySuite) TestPlainClone() {
-	dir, err := os.MkdirTemp("", "")
-	s.NoError(err)
+	dir := "rel-dir"
+	err := os.Mkdir(dir, 0o755)
+	s.Require().NoError(err)
+
+	defer func() {
+		os.RemoveAll(dir)
+	}()
```

Comment:
Instead, we could use `s.T().Cleanup`

Reply:
Good shout - just replaced it accordingly. PTAL